README.md

```diff
@@ -14,6 +14,34 @@
 
 This repo contains Jupyter Notebooks useful for several aspects of Blue Team work.
 
+## Installation
+
+### Linux
+You need Jupyter Notebooks! On linux, run:
+```
+$ sudo apt-get install jupyter
```

Comment:
Jupyter can be installed with `pip`, so probably should use the same method for everthing.

Reply:
Apt-get install replaced with pip